web/src/components/search/MatchList.tsx

```diff
@@ -36,7 +38,7 @@ export const MatchList = (props: Props): ReactElement => {
           <li key={match.filename}>
             {match.cmsCollectionName && (
               <a
-                href={`/mgmt/cms#/collections/${match.cmsCollectionName}/entries/${match.filename}`}
+                href={`/mgmt/cms#/collections/${cmsMap[match.filename]?.[props.collectionLabel] ?? null}/entries/${match.filename}`}
```

Comment:
[pebble] right now, the URL could potentially be `null`. I might change this component to only provide a link if we know know that `cmsMap[match.filename]?.[props.collectionLabel]` is not null.
```js
                href={`/mgmt/cms#/collections/${cmsMap[match.filename]?.[props.collectionLabel] ?? null}/entries/${match.filename}`}
```